server2/Arcadia.Assistant/Arcadia.Assistant.Web/Controllers/DownloadAndroidWebController.cs

```diff
@@ -0,0 +1,35 @@
+﻿namespace Arcadia.Assistant.Web.Controllers
+{
+    using Arcadia.Assistant.AppCenterBuilds.Contracts;
+    using Arcadia.Assistant.MobileBuild.Contracts;
+    using Arcadia.Assistant.Web.Models;
+    using Microsoft.AspNetCore.Mvc;
+    using System.Threading;
+    using System.Threading.Tasks;
+
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public class DownloadAndroidWebController : Controller
+    {
+        private readonly IMobileBuildActorFactory mobileBuildActor;
+        private readonly IHelpSettings helpSettings;
+
+        public DownloadAndroidWebController(IHelpSettings helpSettings, IMobileBuildActorFactory mobileBuildActor)
+        {
+            this.helpSettings = helpSettings;
+            this.mobileBuildActor = mobileBuildActor;
+        }
+
+        [Route("download/android")]
+        [HttpGet]
+        public async Task<IActionResult> Index()
+        {
```

Comment:
I left these comments already. Where are they? What's the reason to create own CTS instead of existing one?

Reply:
Done